pkg/utils/utils.go

```diff
@@ -112,7 +124,16 @@ func DiscoverSriovDevices() ([]sriovnetworkv1.InterfaceExt, error) {
 				}
 			}
 		}
-		pfList = append(pfList, iface)
+
+		for _, devName := range deviceNames {
```

Comment:
In case needed, this can be done in O(1) by checking if  `/sys/class/net/<devName>/device/uevent` exists
wasn't sure what is better, high level approach as the PR have, or optimized implementation.